blogposts/2019/announcing-sourcegraph-3.10.md

```diff
@@ -99,7 +99,7 @@ Video will be added later, not essential for release post, but important in its
 
 With the completion of the Java LSIF indexer, LSIF-based precise code intelligence now supports five languages: Go, Typescript, Java, C++, and Python. See [lsif.dev](https://lsif.dev/) for the most up-to-date information on the list of languages with LSIF indexers.
 
-We are looking for feedback from project owners interested in trying out precise code intelligence! Use our new [LSIF quickstart guide](https://docs.sourcegraph.com/user/code_intelligence/lsif_quickstart) to try it out manually before adding it as a step in your CI.
+We are looking for feedback from project owners interested in trying out precise code intelligence! Use our new [LSIF quickstart guide](https://docs.sourcegraph.com/user/code_intelligence/precise_code_intelligence_quickstart) to try it out manually before adding it as a step in your CI.
```

Comment:
Looks like the old link was correct here. I get a 404 on the new URL.

Reply:
whoops, good catch